mlir/include/mlir/Dialect/SparseTensor/IR/SparseTensorAttrDefs.td

```diff
@@ -438,6 +438,10 @@ class RankedSparseTensorOf<list<Type> allowedTypes>
 
 def AnyRankedSparseTensor : RankedSparseTensorOf<[AnyType]>;
 
+class ScalarLikeOf<list<Type> allowedTypes>
```

Comment:
This is technically no longer a "sparse tensor trait" as defined by the header of this section (so in the long run we may want to promote this to a more general place). But OK for now.

Reply:
Yeah, agree.